docs/resources/nat_dnat_rule.md

```diff
@@ -91,67 +94,68 @@ resource "huaweicloud_nat_dnat_rule" "dnat_rule_pgSql" {
 
 The following arguments are supported:
 
-* `region` - (Optional, String, ForceNew) The region in which to create the dnat rule resource. If omitted, the
-  provider-level region will be used. Changing this creates a new dnat rule.
+* `region` - (Optional, String, ForceNew) Specifies the region where the DNAT rule is located.  
+  If omitted, the provider-level region will be used. Changing this will create a new resource.
 
-* `nat_gateway_id` - (Required, String, ForceNew) ID of the nat gateway this dnat rule belongs to. Changing this creates
-  a new dnat rule.
+* `gateway_id` - (Required, String, ForceNew) Specifies the ID of the NAT gateway to which the DNAT rule belongs.  
+  Changing this will create a new resource.
 
-* `floating_ip_id` - (Required, String, ForceNew) Specifies the ID of the floating IP address. Changing this creates a
-  new dnat rule.
+* `floating_ip_id` - (Required, String, ForceNew) Specifies the ID of the floating IP address.  
```

Comment:
look almost all parameters can be updated, please recheck the doc

Reply:
fixed